modular_citadel/code/datums/status_effects/chems.dm

```diff
@@ -710,15 +741,6 @@
 		deltaResist *= 1.25
 	if (owner.reagents.has_reagent("neurine"))
 		deltaResist *= 1.5
-	if (!HAS_TRAIT(owner, TRAIT_CROCRIN_IMMUNE) && M.canbearoused)
-		if (owner.reagents.has_reagent("anaphro"))
-			deltaResist *= 1.5
-		if (owner.reagents.has_reagent("anaphro+"))
-			deltaResist *= 2
-		if (owner.reagents.has_reagent("aphro"))
-			deltaResist *= 0.75
-		if (owner.reagents.has_reagent("aphro+"))
-			deltaResist *= 0.5
```

Comment:
Is this really necessary to remove? It'd probably be better to add a lewdchem check like the other things that had lewdchem checks added

Reply:
People saying that MKUltra was made purely for lewds wore me down. I wanted to see the mechanics in action as I intended them, but, I've reverted this and also put in the check.